scripts/deploy.sh

```diff
@@ -3,30 +3,52 @@ set -e
 
 source scripts/travis_env.sh
 
+ARCHS=( armv7 x86 arm64_v8a x86_64 )
```

Comment:
the main changes here are this array being added to avoid rewriting everywhere the same thing